frontend/src/language/no.json

```diff
@@ -174,5 +174,15 @@
     "Hours between inspections": "Timer mellom inspeksjoner",
     "Days between inspections": "Dager mellom inspeksjoner",
     "Confirm installation": "Bekreft anlegg",
-    "N": "N"
+    "N": "N",
```

Comment:
Can you change also change from "Ingen inspeksjon planlagt" to "Ingen planlagt inspeksjon"